### PR TITLE
style guide alignment and tidy

### PR DESCRIPTION
**Pull request by aevesdocker**
More detailed tidy of Merge, Version and Name, Secrets, Extensions, Fragments

### CODE REVIEW

File: spec.md [hunk 1]
```diff
@@ -238,21 +238,21 @@ services:
 
 ## Version top-level element
 
-Top-level `version` property is defined by the specification for backward compatibility but is only informative.
+The top-level `version` property is defined by the Compose Specification for backward compatibility. It is only informative.
 
-Compose doesn't use this version to select an exact schema to validate the Compose file, but
-prefer the most recent schema at the time it has been designed.
+Compose doesn't use `version` to select an exact schema to validate the Compose file, but
+prefers the most recent schema at the time it has been designed.
```
Comment (usha-mandya) [inline spec.md:244]: Same comment as above on simplifying the last part

File: spec.md [hunk 2]
```diff
@@ -2424,22 +2433,17 @@ secrets:
...
-
-Compose file needs to explicitly grant access to the secrets to relevant services in the application.
-
+If `external` is set to `true`, all other attributes apart from `name` are irrelevant. If Compose detects any other attribute, it rejects the Compose file as invalid.
 
+Your Compose file needs to explicitly grant access to the secrets to relevant services in your application.
 ## Fragments
 
-With Docker Compose you can use built-in [YAML](http://www.yaml.org/spec/1.2/spec.html#id2765878) features to make your Compose file neater and more efficient. Anchors and aliases let you create re-usable blocks. This is useful if you start to find common configurations that span multiple services. Having re-usable blocks minimizes potential mistakes.
+With Compose you can use built-in [YAML](http://www.yaml.org/spec/1.2/spec.html#id2765878) features to make your Compose file neater and more efficient. Anchors and aliases let you create re-usable blocks. This is useful if you start to find common configurations that span multiple services. Having re-usable blocks minimizes potential mistakes.
```
Comment (usha-mandya) [inline spec.md:2441]: nit

```suggestion
With Compose, you can use built-in [YAML](http://www.yaml.org/spec/1.2/spec.html#id2765878) features to make your Compose file neater and more efficient. Anchors and aliases let you create re-usable blocks. This is useful if you start to find common configurations that span multiple services. Having re-usable blocks minimizes potential mistakes.
```

**Review by usha-mandya — APPROVED** [review]
Looks great. Made some minor suggestion.

**Comment by aevesdocker** [comment]
Ready for review @glours @ndeloof when you get a spare moment. Should be a quick/straightforward one - I've not done anything majorly radical 🤞

**Review by ndeloof — APPROVED** [review]

File: 11-extension.md [hunk 3]
```diff
@@ -86,12 +86,12 @@ services:
      https_proxy: $https_proxy
 ```
 
-The `nodeinfo` and `echoit` services both use merge it in, then set their specific image and environment. 
+The `nodeinfo` and `echoit` services both use the `x-function` extension, then set their specific image and environment.
```
Comment (glours) [inline 11-extension.md:89]:
```suggestion
The `nodeinfo` and `echoit` services both include the `x-function` extension via the `&function` anchor, then set their specific image and environment. 
```

File: 12-interpolation.md [hunk 4]
```diff
@@ -23,8 +23,8 @@ Interpolation can also be nested:
 - `${VARIABLE?$FOO}`
 - `${VARIABLE:-${FOO:-default}}`
 
-Other extended shell-style features, such as `${VARIABLE/foo/bar}`, are not
-supported by the Compose specification.
+Other extended shell-style features, such as `${VARIABLE/foo/bar}`, is not
```
Comment (glours) [inline 12-interpolation.md:26]: Should be conjugated with `features`, no?
```suggestion
Other extended shell-style features, such as `${VARIABLE/foo/bar}`, are not
```

**Review by glours — COMMENTED** [review]
I saw 2 things to check before merging the PR

File: 12-interpolation.md (hunk 4 above)
Comment (aevesdocker) [reply 12-interpolation.md]: Woops! Yes!

**Review by glours — APPROVED** [review]
LGTM

**Pull request merged**